media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -32,7 +32,7 @@
 #include <sys/ioctl.h>
 #include <unistd.h>
 
-#ifndef ANDROID
+#if !defined(ANDROID) && defined(X11_FOUND)
```

Comment:
Shouldn't we use just:
`#ifdef X11_FOUND`
and assure that build system don't set this for Android path? Smaller number of defines and code, the better.